curriculum/challenges/english/08-coding-interview-prep/data-structures/remove-from-a-set.english.md

```diff
@@ -6,12 +6,12 @@ challengeType: 1
 
 ## Description
 <section id='description'>
-In this exercises we are going to create a delete function for our set. The function should be named <code>this.remove</code>. This function should accept a value and check if it exists in the set. If it does, remove that value from the set, and return true. Otherwise, return false.
+In this exercises we are going to create a delete function for our set.
 </section>
 
 ## Instructions
 <section id='instructions'>
-
+The function should be named <code>this.remove</code>. This function should accept a value and check if it exists in the set. If it does, remove that value from the set, and return true. Otherwise, return false.
```

Comment:
@ezioda004 over here

Reply:
This is where @cmccormack asked you to change the code tags to markdown style

Reply:
Ok - but why markdown? `<code> </code>` is semantically correct element. here - more accessible than the markdown.

Reply:
My understanding was we were to use GFM instead of raw html wherever possible; if this is not the case, then no worries.

Reply:
Yep, thats true but only for "guides" articles. For challenges, we follow [challenge formatting](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-curriculum-challenges.md#formatting-challenge-text) guide.

Reply:
Interesting, looks like those were written two years ago, I wonder if its out of date or still the standard for challenges.  I don't see why they would be different but we can get clarification outside of this PR.  Looks good to me 👍